Strategies/ITVision.md

```diff
@@ -0,0 +1,134 @@
+# ESDC's IT Vision
+
+[HDLC](HumanDevelopmentLifeCycle.md) -&gt; [KPIs](IITB_KPIs.md) -&gt; IT Vision -&gt; Guiding Principles -&gt; IT Strategies (roadmap)
+
+The following is ESDC's IT Vision.
+It paints a vision of the future of IT within ESDC.
+It is **not** meant to be time bound, but rather as an ideal state to which ESDC IT needs to strive to get to.
+Such state will continuously evolve when new information and knowledge is captured, re-adjusting the vision in the process.
+
+The IT Vision complements the [HDLC](HumanDevelopmentLifeCycle.md) story which is taken from the point of view of the citizen, and the [KPIs](IITB_KPIs.md) which define the performance metrics needed to sustain the HDLC, by providing the point of view of the organization's IT capabilities.
+
+The IT Vision leverages the work done in the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), ensures alignment with [IITB's high level priorities](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml), and brings the [GC's Strategic Plan for IM/IT 2017-2021](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) into ESDC's operational context.
+
+The ESDC IT Vision is explained using the [GC's Strategic Plan for IM/IT 2017-2021](https://www.canada.ca/en/treasury-board-secretariat/services/information-technology/strategic-plan-2017-2021.html#toc4) six vision pillars:
+
+1. Digital Services
+2. Open and accessible
+3. Security
+4. Enterprise management
+5. Community
+6. Workplace
+
+## 1. Digital Services
+
+All of [ESDC's services](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) are delivered through technology with a large majority running on the public cloud to seek benefits of rapid elasticity, broad internet access, and up to date security patching.
+
+Artificial Intelligence is being used to drastically [reduce toil](https://landing.google.com/sre/sre-book/chapters/eliminating-toil/), enable auto-enrolment by design, provide insights for service innovation, while ensuring ethical use of [automated decision-making](https://tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592). Artificial Intelligence is a shared expertise between the Chief Information Officer (CIO, responsible for the technology), and the Chief Data Officer (CDO, responsible for data scientists and management).
+
+User Experiences (websites, web applications, and mobile applications) is **rarely** provided by ESDC.
```

Comment:
I'll reword, by User Experience I was referencing to front end development.
In the end state, the core of ESDC's IT will focus on services, delivered via APIs. The interactions with those services would not be a core business of ESDC. This is where the industry would be best (or even the public).

principle: do not compete with private sector

Reply:
I agree and support the idea of weaving our services to Canadians through the entire ecosystem but I thing there's still a place for a Government facing interaction, as long as it's through the OneGC vision. The API first still supports that possiblity where we don't create extra front facing interfaces but have our services leveraged through the OneGC provided interface.

Reply:
@gcharest see commit #973674c
reworded the paragraph. Added reference to OneGC for user interfaces.

resolve conversation?